earth_enterprise/src/common/generic_utils.h

```diff
@@ -16,7 +16,8 @@
 
 #ifndef GEO_EARTH_ENTERPRISE_SRC_COMMON_GENERIC_UTILS_H_
 #define GEO_EARTH_ENTERPRISE_SRC_COMMON_GENERIC_UTILS_H_
-
+#include <string>
```

Comment:
Are these necessary? I don't see any other changes in this file.

Reply:
Holdover from a utility function I added but then deleted. Addressed